string info
uint64 time_stamp
uint8[] camera_raw
uint8[] camera_jpg
